packages/styles/scss/patterns/cards-with-images/_cards-with-images-group.scss

```diff
@@ -0,0 +1,116 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+@import '../../components/link-with-icon/link-with-icon';
+
+@mixin themed-items {
+  .bx--card-link {
+    background-color: $ui-01;
+    border-color: $ui-02;
+    outline: 1px solid $ui-background;
```

Comment:
can you overwrite the padding from the card link component by adding:
` padding: 0;` here?

Reply:
Okay.